tests/Phinx/Db/Adapter/SQLiteAdapterTest.php

```diff
@@ -1161,6 +1161,37 @@ public function testQueryBuilder()
         $this->assertEquals(1, $stm->rowCount());
     }
 
+    public function testQueryWithParams()
+    {
+        $table = new \Phinx\Db\Table('table1', [], $this->adapter);
+        $table->addColumn('string_col', 'string')
+            ->addColumn('int_col', 'integer')
+            ->save();
+
+        $this->adapter->insert($table->getTable(), [
+            'string_col' => 'test data',
+            'int_col' => 10,
+        ]);
+
+        $this->adapter->insert($table->getTable(), [
+            'string_col' => null,
+        ]);
+
+        $this->adapter->insert($table->getTable(), [
+            'int_col' => 23,
+        ]);
+
+        $countQuery = $this->adapter->query('SELECT COUNT(*) as c FROM table1 WHERE int_col > ?', [5]);
```

Comment:
&lt;pedantism>You've got a lowercase `as` but other keywords are upper.&lt;/pedantism>